test.md

```diff
@@ -0,0 +1 @@
+**This file should be ignored by git**
```

Comment:
- [ ] Kindly make the `test.md` file is not represented in the repo as it is already ignored by git.

Reply:
Hello, @levy002. I added the message "This file should be ignored by git" to the test.md file as per instructed in the project requirements. I'm not 100% clear on your instructions. Mind explaining?